content/projects/git-exercises/_index.md

```diff
@@ -341,7 +341,7 @@ Go look at your network page. each of the dots there represent commits you made.
 2. Now let's clone a repo. point your browser here: https://github.com/Umuzi-org/a-repo-to-clone
 3. Now there is a friendly green button that says "Clone or download". Click on it.
 4. You will see a url come up. Copy it. You will need to paste it into the terminal in a moment
-5. In your terminal type in `git clone $THE_URL_YOU_JUST_COPIED`. It should look something like this: `git clone https://github.com/Umuzi-org/tech-department`
+5. In your terminal type in `git clone $THE_URL_YOU_JUST_COPIED`. It should look something like this: `git clone https://github.com/Umuzi-org/a-repo-to-clone`
 6. `cd` into the tech-department directory that was just created
```

Comment:
Please also correct this line to refer to the corrected directory

Reply:
This is fixed